src/Cache/Signature.php

```diff
@@ -78,7 +78,7 @@ private static function utf8Encode(array $data): array
     {
         array_walk_recursive($data, static function (&$item): void {
             if (\is_string($item) && !mb_detect_encoding($item, 'utf-8', true)) {
-                $item = utf8_encode($item);
+                $item = base64_encode($item);
```

Comment:
I think this will invalidate the cache on the first run with this change. IMO it's fine though.

Maybe we can rename the method to something more suitable, like `makeJsonCompatible()`?

Reply:
> I think this will invalidate the cache on the first run with this change.

Most users probably do not have not-UTF8-encoded configs, and the cache is invalidated with every version bump anyway.

> Maybe we can rename the method to something more suitable, like `makeJsonCompatible()`?

Changed it to `makeJsonEncodable`, hope that's OK :)